### PR TITLE
Add min-width to input-group-btn

### PR DESCRIPTION
**Pull request by OFFLlNE**

### CODE REVIEW

**Comment by oliverviljamaa** [comment]
I'm not sure about the complications here:
1) No `.less` file seems to be changed, did you change the `.css` file manually or am I missing something?
2) Surely small and large selects should have different minimum widths
3) Isn't this change only necessary for our _amount currency select_, therefore adding it to `input-group-btn` is too much?

**Comment by StevePole** [comment]
I agree with Oliver's last point.  On 1, if no CSS was updated you need to run `grunt` and also commit the changed files.

File: less/input-groups.less [hunk 1]
```diff
@@ -126,6 +126,15 @@
   vertical-align: middle; // Match the inputs
 }
 
+.input-group-currency-select-btn {
+  min-width: 144px;
+}
+@media (max-width: @screen-xs-max) {
```
Comment (oliverviljamaa) [inline less/input-groups.less]: Could we do the media query first instead?

Comment (oliverviljamaa) [inline less/input-groups.less]: Could we do the media query mobile first instead?

Comment (OFFLlNE) [reply less/input-groups.less]: As in having media query before line 129?

File: less/input-groups.less [hunk 2]
```diff
@@ -126,6 +126,15 @@
   vertical-align: middle; // Match the inputs
 }
 
+.input-group-currency-select-btn {
```
Comment (oliverviljamaa) [inline less/input-groups.less]: Is this the class we need to add to the amount currency select? If yes, do we have a corresponding PR. Also, as it's specific to _amount-currency-select_, couldn't the name reflect that? And potentially also the file?

File: less/input-groups.less (hunk 1 above)
Comment (oliverviljamaa) [reply less/input-groups.less]: Nah, as in having mobile as the default behaviour and using `min-width` media query to change appearance for wider viewports.

File: less/input-groups.less (hunk 2 above)
Comment (OFFLlNE) [reply less/input-groups.less]: I wanted to make changes in bootstrap before making a PR for component, will do a PR for the component as well then ;)
did not understand what do you mean by the file reflecting the name?

Comment (oliverviljamaa) [reply less/input-groups.less]: Do amount currency selects have any other custom styles? If yes, where are they? If no, I think it'd be more obvious and clear if these styles were in a file called `amount-currency-select.less` etc.

Comment (OFFLlNE) [reply less/input-groups.less]: taking it out the `input-group.less` and creating a separate less?

**Review by StevePole — APPROVED** [review]

File: less/amount-currency-select.less [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.amount-currency-select-btn {
```
Comment (oliverviljamaa) [inline less/amount-currency-select.less:1]: Literally last comment, sorry...

Could we have the media query nested inside the `.amount-currency-select-btn` selector? One potential mistake point less.

File: less/bootstrap.less [hunk 4]
```diff
@@ -43,6 +43,7 @@
 @import "close.less";
 @import "loaders.less";
 @import "tiles.less";
+@import "amount-currency-select";
```
Comment (oliverviljamaa) [inline less/bootstrap.less]: For consistency, I think it'd be nicer to include the extension.

Comment (OFFLlNE) [reply less/bootstrap.less]: yup, just noticed myself too :D

File: less/amount-currency-select.less (hunk 3 above)
Comment (OFFLlNE) [reply less/amount-currency-select.less:1]: no worries, love comments as long as I know what to do :D 
Waiting for that push force to come up here

Comment (OFFLlNE) [reply less/amount-currency-select.less:1]: Suprised that grunt did not update the bootstrap of that change, does it matter?

Comment (oliverviljamaa) [reply less/amount-currency-select.less:1]: @OFFLlNE It's fine, a media query in a selector is not allowed in CSS, but is (and often is preferred) in LESS. Therefore, it compiles to the same thing (media query outside the selector).

**Review by oliverviljamaa — APPROVED** [review]

**Pull request merged**